public/admin/config.yml

```diff
@@ -255,3 +255,86 @@ collections:
         file: src/cms-content/learn/footer.json
         fields:
           - { label: Body, name: body, widget: markdown, default: true }
+      - name: Case Studies
+        file: 'src/cms-content/learn/case-studies.json'
+        label: Case Studies
+        fields:
+          - label: Title
+            name: title
+            widget: text
+            hint: Title for the case studies landing page
+          - label: Intro
+            name: intro
+            widget: markdown
+            hint: Introduction for the case studies landing page
+          - label: Categories
+            name: categories
+            widget: list
+            fields:
+              - {
+                  label: Category Name,
+                  name: title,
+                  widget: text,
+                  hint: Name of the category,
+                }
+              - {
+                  label: Anchor Link ID,
+                  name: anchorId,
+                  widget: string,
+                  hint: 'String to use for anchor links (ex: public-health)',
```

Comment:
same as comment above. would nix dev-language (ie. string, anchor), and just write something like 'ID to use in URLs linking to this category'

Reply:
or rather, maybe the hint should just refer to the format and not the use. ie 'all lowercase category name, divided by dashes if multi-word'